### ISSUE TITLE
csysdig segfaulted whilst examining scap file

### ISSUE THREAD

**Issue by aspiers** [draios/sysdig#503]
This happened with the `dev` branch.  I can reproduce as follows:

- launch `csysdig`
- hit Enter to select a process
- hit F8 for actions
- `<NO ACTIONS>` is displayed
- hit Enter
- boom! segfault

I can provide the `foo.scap` if the following isn't sufficient information.

```
# gdb =csysdig core
GNU gdb (GDB; openSUSE 13.2) 7.8
Copyright (C) 2014 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-suse-linux".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://bugs.opensuse.org/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word"...

warning: /etc/gdbinit.d/gdb-heap.py: No such file or directory
Reading symbols from /usr/bin/csysdig...Reading symbols from /usr/lib/debug/usr/bin/csysdig.debug...done.
done.
[New LWP 14691]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib64/libthread_db.so.1".
Core was generated by `csysdig -r foo.scap'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  _M_data (this=<optimized out>) at /usr/include/c++/4.8/bits/basic_string.h:293
293           { return  _M_dataplus._M_p; }
(gdb) bt
#0  _M_data (this=<optimized out>) at /usr/include/c++/4.8/bits/basic_string.h:293
#1  _M_rep (this=<optimized out>) at /usr/include/c++/4.8/bits/basic_string.h:301
#2  size (this=<optimized out>) at /usr/include/c++/4.8/bits/basic_string.h:716
#3  sinsp_cursesui::run_action (this=this@entry=0x7ffc963ee810, action=0x0)
    at /usr/src/debug/sysdig-0.5.1+git.1449971589.028d205/userspace/libsinsp/cursesui.cpp:2205
#4  0x000000000046161e in sinsp_cursesui::handle_input (this=this@entry=0x7ffc963ee810, ch=<optimized out>)
    at /usr/src/debug/sysdig-0.5.1+git.1449971589.028d205/userspace/libsinsp/cursesui.cpp:1840
#5  0x000000000043ce62 in process_event (next_res=6, evt=0x1189658, this=0x7ffc963ee810)
    at /usr/src/debug/sysdig-0.5.1+git.1449971589.028d205/userspace/libsinsp/cursesui.h:424
#6  do_inspect (inspector=inspector@entry=0x1189620, cnt=cnt@entry=18446744073709551615, ui=ui@entry=0x7ffc963ee810)
    at /usr/src/debug/sysdig-0.5.1+git.1449971589.028d205/userspace/sysdig/csysdig.cpp:196
#7  0x000000000043eb11 in csysdig_init (argc=<optimized out>, argv=<optimized out>)
    at /usr/src/debug/sysdig-0.5.1+git.1449971589.028d205/userspace/sysdig/csysdig.cpp:592
#8  0x000000000042a4a7 in main (argc=<optimized out>, argv=<optimized out>)
    at /usr/src/debug/sysdig-0.5.1+git.1449971589.028d205/userspace/sysdig/csysdig.cpp:645
```

**Comment by ldegio** [comment]
this should be fixed by 09193e3a1f2b58217e7e5aea9fe7925dd9b90f19

**Comment by aspiers** [comment]
Awesome, thanks! :)
